html/semantics/document-metadata/the-base-element/base-data.html

```diff
@@ -0,0 +1,32 @@
+<!-- Please update base-javascript.html together with this -->
```

Comment:
Does having a comment instead of doctype on the first line put this into quirks mode?

Reply:
No it doesn't. In IE6 it did.